core/types/transaction.go

```diff
@@ -1193,11 +1135,44 @@ func (tx *Transaction) AsMessageWithSender(s Signer, baseFee *big.Int, sender *c
 	return msg, err
 }
 
+// CompareFee compares new fee and the tx.Fee() two transactions and returns the output of
+// bigInt Cmp method
+func (tx *Transaction) CompareFee(newFee *big.Int) int {
+	if newFee == nil {
+		return -1 // cannot be compared, so return less than
+	}
+	oldTxFee := tx.GasPrice()
+	return oldTxFee.Cmp(newFee)
+}
+
+func (tx *Transaction) CompareGivenBaseFee(newTx *Transaction, baseFee *big.Int) int {
+	return 0
+}
+
+func CompareFeeBetweenTx(a, b *Transaction) int {
+	aTxFee := a.GasPrice()
+	bTxFee := b.GasPrice()
+	// check if the Fee put in the tx is within this desired bitInt limit
+	if aTxFee.BitLen() < 0 || aTxFee.BitLen() > 256 {
+		return 1
+	}
+	if bTxFee.BitLen() < 0 || bTxFee.BitLen() > 256 {
+		return -1
```

Comment:
Is there a reason to return 1 or -1 here? Why not return an error?

Reply:
this is how it was done before

Reply:
Can you link to an example?

Reply:
This is not needed, deleted this